templates/default/jolokia.policy.erb

```diff
@@ -4,9 +4,14 @@
   <http>
     <method>get</method>
   </http>
+  <http>
+    <method>post</method>
+  </http>
 
   <commands>
     <command>read</command>
     <command>list</command>
+    <command>version</command>
```

Comment:
why are the version and search needed here?
@purushottambaghel

Reply:
Actually not needed as of now, So removing it